pytorch_lightning/loggers/wandb.py

```diff
@@ -33,6 +33,7 @@
 
 _WANDB_AVAILABLE = _module_available("wandb")
 _WANDB_GREATER_EQUAL_0_10_22 = _compare_version("wandb", operator.ge, "0.10.22")
```

Comment:
nit: Should we add these to `utilities/imports.py` (@awaelchli) ?

Reply:
I disagree with this suggestion.

Why wouldn't the WandB file define its own available check?
Doing so keeps all wandb things together and avoids one dependency between files.
No other file should need to import this variable since the code is all condensed into this logger.

Reply:
well, I thought we keep all the imports in one place. like JSONARGPARSE_AVAILABLE,  which is used only for cli, Neptune_Available (another loggeR), BAGUA_AVAILABLE, ... etc.

Reply:
For legacy, IMO it would be cleaner to move as many as possible out

Similarly to my comment here https://github.com/PyTorchLightning/pytorch-lightning/pull/11797#discussion_r802963748

That'd included those 4 you mentioned too